client/components/TestRun/index.jsx

```diff
@@ -192,14 +209,97 @@ class TestRun extends Component {
+                                                ) {
+                                                    resultClassName =
+                                                        'conflicts';
+                                                } else if (
+                                                    testersResult.status ===
+                                                    'complete'
+                                                ) {
+                                                    resultClassName =
+                                                        'complete';
+                                                }
+                                            } else if (
+                                                this.state.currentTestIndex -
+                                                    1 >
+                                                i
+                                            ) {
+                                                resultClassName = 'skipped';
+                                            }
+                                            return (
+                                                <li
+                                                    className={`test-name-wrapper ${resultClassName}`}
+                                                >
+                                                    <span className="progress-indicator"></span>
+                                                    <a
+                                                        href="#"
+                                                        onClick={e => {
+                                                            this.displayTestByIndex(
+                                                                i + 1
+                                                            );
+                                                        }}
+                                                        className="test-name"
+                                                    >
+                                                        {t.name}
+                                                    </a>
+                                                </li>
+                                            );
+                                        })}
+                                    </ol>
+                                </>
+                            ) : (
+                                <button
+                                    onClick={this.toggleTestNavigator}
+                                    className="test-navigator-toggle"
+                                >
+                                    Show Test Navigator
+                                </button>
+                            )}
+                        </aside>
+                        <button
+                            onClick={this.toggleTestNavigator}
+                            className="test-navigator-toggle show"
```

Comment:
@isaacdurazo - The way I did "show" looks weird. I cannot tell if it is the `css` or my implentation.